webapi/src/main/scala/org/knora/webapi/responders/v1/UsersResponderV1.scala

```diff
@@ -101,57 +91,320 @@ class UsersResponderV1 extends ResponderV1 {
       * @param username the username of the user.
       * @return a [[UserProfileV1]] describing the user.
       */
-    private def getUserProfileByUsernameV1(username: String): Future[UserProfileV1] = {
+    private def getUserProfileByUsernameV1(username: String, profileType: UserProfileType.Value): Future[UserProfileV1] = {
+        // TODO: add caching of user profiles that was removed from [[Authenticator]]
```

Comment:
Has this been done already? If so, please remove the comment. If not, make an issue for it on GitHub.